django/contrib/auth/management/__init__.py

```diff
@@ -37,6 +37,11 @@ def create_permissions(app_config, verbosity=2, interactive=True, using=DEFAULT_
     if not app_config.models_module:
         return
 
+    # It's important to ensure here that create_contenttypes has been called for
+    # this app (we don't control post_migrate signal receiver's ordering).
+    from django.contrib.contenttypes.management import create_contenttypes
```

Comment:
Does it break anything if this is moved to the top? I would assume this module can safely depend on the later?